mmv1/products/bigquery/api.yaml

```diff
@@ -112,6 +112,31 @@ objects:
                     A-Z), numbers (0-9), or underscores (_). The maximum length
                     is 1,024 characters.
                   required: true
+            - !ruby/object:Api::Type::NestedObject
+              name: 'dataset'
+              description: |
+                Grants all resources of particular types in a particular dataset read access to the current dataset.
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: 'dataset'
```

Comment:
Is this one required? It feels like it should beat a glance.

Reply:
Yup!